app/uk/gov/hmrc/trackingconsentfrontend/assets/src/common/pageHandler.ts

```diff
@@ -5,13 +5,13 @@ import { enableTrackingConsent } from "../constants/featureNames";
 import { SERVICE_PAGE_LOAD_EVENT } from "../constants/events";
 import getGtmContainerId from "./getGtmContainerId";
 
-const pageHandler = (document: HTMLDocument, userPreference: UserPreferences, pageRenderer, containerId) => {
+const pageHandler = (document: HTMLDocument, userPreference: UserPreferences, pageRenderer) => {
     const gtmContainerId = getGtmContainerId()
     if (gtmContainerId === undefined) {
-        console.warn("Deprecation notice: container id not specified in data-gtm-container attribute. The functionality you are currently using will be removed on 03 November 2020")
+        console.error("container id not specified in data-gtm-container attribute. The previous default container ID was removed on 03 November 2020")
```

Comment:
I think we should be stopping execution here - maybe a `throw new Error` or maybe `return`/`else` after the console error.

Reply:
@natcarey That seems fair enough - would you also keep in the thrown error within `enableGtm` if the container id is undefined?

Reply:
@natcarey If we error here, then this would cause an error even before the feature is enable - is that definitely what we want?

Reply:
Hmm, I guess the actual error here is that GTM can't be included, so maybe it's just about putting `enableGtm(gtmContainerId)` into an `else` block would be the best representation of that.  The logic there is "if you provide a GTM container reference we'll include GTM".

Reply:
I would go with throwing in pageHandler as Nat suggests because it doesn't make sense to continue without gtm.